internal/testrunner/coverageoutput_test.go

```diff
@@ -0,0 +1,279 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package testrunner
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCoberturaCoverage_Merge(t *testing.T) {
+	tests := []struct {
+		name               string
+		rhs, lhs, expected CoberturaCoverage
+		wantErr            bool
+	}{
+		{
+			name: "merge sources",
```

Comment:
nit: I'm thinking now if it isn't simpler and easier to read to load these tests from a `testdata` directory. WDYT?